e2e/playwright/onboarding-tests.spec.ts

```diff
@@ -21,58 +21,54 @@ import { expectPixelColor } from './fixtures/sceneFixture'
 // we must set it to empty for the tests where we want to see the onboarding immediately.
 
 test.describe('Onboarding tests', () => {
-  test(
-    'Onboarding code is shown in the editor',
-    {
-      appSettings: {
-        app: {
-          onboarding_status: '',
-        },
+  test('Onboarding code is shown in the editor', async ({
+    page,
+    homePage,
+    tronApp,
+  }) => {
+    if (!tronApp) {
+      fail()
+    }
```

Comment:
Does this add a step in the process of switching a test from "electron mode" to "browser mode"? Can we pass a message into `fail()` that can alert the developer in this situation that `'tronApp was not found'` or something?

Reply:
It appears we can https://playwright.dev/docs/api/class-test#test-fail but unfortunately tronApp is optional so we can't just leave it out in some cases and in, in others, otherwise Playwright will complain and not run at all. 

We could do a follow up maybe.

In those tests though, tronApp is a necessary component, so if it's falsey / undefined, it should really fail immediately